.github/workflows/docker.yml

```diff
@@ -10,4 +13,15 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Build the Docker image
-        run: docker build -t docs-rs -f dockerfiles/Dockerfile .
+        run: docker build -t docs-rs -f dockerfiles/Dockerfile . --target=web-server
```

Comment:
What does `--target web-server` do?

Reply:
So this change would remove the validation of the main docker image from this job

Reply:
> What does --target web-server do?

The Dockerfile has various stages that it can build based on what you want the image to do. I took this directly from the [`deploy.yml`](https://github.com/rust-lang/docs.rs/blob/master/.github/workflows/deploy.yml) which currently does deploying of the production image to ecr for the production environment.